api/routes/dishesRouter.js

```diff
@@ -0,0 +1,118 @@
+const express = require('express')
+
+const dishesModel = require('../models/dishesModel.js')
+
+const router = express.Router()
+
+router.use(express.json())
+
+router.get('/', async (req, res) => {
+    try {
+        const dishes = await dishesModel.find()
+        res.status(200).json(dishes)
+    } catch (error) {
+        console.log(error)
+    }
+})
+
+router.get('/:id',  validateDishId, async (req, res) => {
+    try {
+        const dish = await dishesModel.findById(req.dish.id)
+        res.status(200).json(dish)
+    } catch (error) {
+        console.log(error)
+        res.status(500).json(error)
+    }
+})
+
+router.get('/:id/recipes',  validateDishId, async (req, res) => {
+    try {
+        const dish = await dishesModel.findById(req.dish.id)
+        const recipes = await dishesModel.findRecipesById(req.dish.id)
+        const recipesList = {

```

Comment:
Ah, cool you took care of the problem I was talking about above here, good job. But I think if we look at this project from the FE perspective this structure is a little funky, if I'm getting a dish by id I want the associated recipes.